packages/lib/slugify.ts

```diff
@@ -5,10 +5,9 @@ export const slugify = (str: string) => {
     .normalize("NFD") // Normalize to decomposed form for handling accents
     .replace(/\p{Diacritic}/gu, "") // Remove any diacritics (accents) from characters
     .replace(/[^\p{L}\p{N}\p{Zs}\p{Emoji}]+/gu, "-") // Replace any non-alphanumeric characters (including Unicode) with a dash
-    .replace(/[\s_]+/g, "-") // Replace whitespace and underscores with a single dash
+    .replace(/[\s_#]+/g, "-") // Replace whitespace, # and underscores with a single dash
     .replace(/^-+/, "") // Remove dashes from start
-    .replace(/-+$/, "") // Remove dashes from end
-    .replace(/\+/g, "-"); // Transform all + to -
```

Comment:
This was not required as + was already being replaced as part of 'Replace any non-alphanumeric characters (including Unicode) with a dash' regex.
Have verified and added tests for it.